### ISSUE TITLE
Training and testing images to VAE are inconsistent? 

### ISSUE THREAD

**Issue by gitformer** [kxhit/zero123-hf#10]
Hi there, 

I find that the training images to VAE are scaled to [-1, 1] and then processed by a torchvision.transforms.Normalization([0.5], [0.5]): [https://github.com/kxhit/zero123-hf/blob/master/train_zero1to3.py#L692](https://github.com/kxhit/zero123-hf/blob/master/train_zero1to3.py#L692)

However, the testing images to VAE are only scaled to [-1, 1]: [https://github.com/kxhit/zero123-hf/blob/master/pipeline_zero1to3.py#L651](https://github.com/kxhit/zero123-hf/blob/master/pipeline_zero1to3.py#L651)

So Is there a discrepancy between training inputs and testing inputs? Thanks!

**Comment by kxhit** [comment]
Hi, 

Both training and testing images to VAE are scaled to [-1, 1].
For testing images to VAE, if they are already torch.tensor (preprocess by torchvision), it will go through: 
[if isinstance(image, torch.Tensor):](https://github.com/kxhit/zero123-hf/blob/6316befb8ee8d78bc36ffc0d98ed91a99ef3cbe1/pipeline_zero1to3.py#L628)
instead L651.

Hope it is clear. Cheers.
